### ISSUE TITLE
srss.cbush_stress is not supported

### ISSUE THREAD

**Issue by AlejandroStewart** [SteveDoyle2/pyNastran#660]
Hello I just found a typo in pyNastran\op2\tables\oes_stressStrain\oes.py line:1225
where it says **_'srss.'_** sould be **_'srss'_** .

Thank you!

**Comment by AlejandroStewart** [comment]
I think it is a bit more complex, some places i see srss and some others srss. (with dot)

**Comment by SteveDoyle2** [comment]
What's the issue/error with it?

The dot prefix sends it to the op2_results.srss object, so
self.op2_results.srss.cquad4_stress of whatever ends up getting built.

**Comment by AlejandroStewart** [comment]
The problem comes when there are accelerations and stresses. 
with prefix='srss.' :
    **_`RuntimeError: result='srss.cbush_stress' is invalid; the name changed or it's a typo.`_**
with prefix='srss':
    No errors, but only accelerations are loaded on the op2 object
with prefix='srss_':
    No errors, but only accelerations are loaded on the op2 object (which is weird)

**Comment by AlejandroStewart** [comment]
Hello, amazing job fixing this! The op2 I was working with now works fine, but the dummy one seems to have problems in line 1224 as the thermal bit does not match and the `self.code_information()` call I think should be `op2.code_information()`

[shock_dummy.zip](https://github.com/SteveDoyle2/pyNastran/files/7182629/shock_dummy.zip)

**Comment by SteveDoyle2** [comment]
I'm a bit confused on the prefix='srss_' part, but I think that's just
being out of date.

I recently did a bit of work on making ABS/NRL/SRSS work a lot better.
Some of those things I added so long ago that some of them had broken over
time.  There was no SRSS support for stress/strain

That said the scaled response spectra support is still poor.  I was really
strict on it in the code how I was supporting it and loosened that up a
bit.  Now to figure out what's broken in the tests that don't fail on my
computer :/
